azkaban-web-server/src/main/java/azkaban/webapp/servlet/ScheduleServlet.java

```diff
@@ -591,6 +593,24 @@ private void ajaxScheduleCronFlow(final HttpServletRequest req,
       return;
     }
 
+    final boolean hasFlowTrigger;
+    try {
+      hasFlowTrigger = this.projectManager.hasFlowTrigger(project, flow);
+    } catch (final Exception ex) {
+      logger.error(ex);
+      ret.put("status", "error");
+      ret.put("message", String.format("Flow %s.%s cannot be scheduled: %s", projectName,
+          flowName, ex));
+      return;
+    }
+
+    if (hasFlowTrigger) {
+      ret.put("status", "error");
+      ret.put("message", String.format("Flow %s.%s is already associated with flow trigger, "
```

Comment:
Is it possible that it is normal that this flow doesn't have flow trigger? Would it be an error given this case? Don't understand what the error message mean here?

Reply:
updated